packages/renderless/src/fluent-editor/index.ts

```
@@ -666,6 +666,106 @@ export const uploadImageToSev =
+                    }
+                  }
+                  findLines(cellBlot)
+                  
+                  // 对每个行应用对齐格式
+                  lines.forEach((line) => {
+                    try {
+                      const lineIndex = state.quill.getIndex(line)
+                      if (lineIndex !== null && lineIndex >= 0) {
+                        const lineLength = line.length()
+                        if (lineLength > 0) {
+                          state.quill.formatText(lineIndex, lineLength, 'align', value, FluentEditor.sources.USER)
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)